op-node/rollup/derive/kromampt_upgrade_transactions.go

```diff
@@ -0,0 +1,171 @@
+)
+
+// init overrides variables defined in the Kroma MPT upgrade.
+func init() {
+}
+
+func KromaMPTNetworkUpgradeTransactions() ([]hexutil.Bytes, error) {
+	upgradeTxns := make([]hexutil.Bytes, 0, 8)
+
+	deployL1BlockTransaction, err := types.NewTx(&types.DepositTx{
+		SourceHash:          deployL1BlockMPTSource.SourceHash(),
+		From:                L1BlockDeployerAddress,
+		To:                  nil,
+		Mint:                big.NewInt(0),
+		Value:               big.NewInt(0),
+		Gas:                 375_000,
```

Comment:
How is this value calculated?

Reply:
Referenced the deployL1BlockTransaction gas value from the ecotone upgrade in the OP.

Reply:
updated